### PR TITLE
generating bulk index mapping based on site

### PR DESCRIPTION
**Pull request by holayeah**
Before the bulk index mapping was being specified in the app config.

Now, The bulk index mapping is being generated dynamically basing on the sensors for a given site.

### CODE REVIEW

File: api.py [hunk 1]
```diff
@@ -176,36 +175,83 @@ def bulk():
...
 
 
+
+def get_associated_sensors(site):
+    """
+    Queryies and returns associated sensors for a given
+    site
+    """
+    associated_sensors = []
+    sensor_table_names = ['seshdash_sensor_bmv', 'seshdash_sensor_emontx', 'seshdash_sensor_emonth']
```
Comment (jlark) [inline api.py]: use seshdash_sensor_mapping table from the dashboard to get the table names.

to get the table type query the `select sensor_type from sensor_mapping where site_id = g.site_id`

https://github.com/GreatLakesEnergy/sesh-dash-beta/blob/master/seshdash/models.py

then based on the node_id you'll lookup the table name

File: api.py [hunk 2]
```diff
@@ -176,36 +175,94 @@ def bulk():
...
+        associated_sensors = associated_sensors  + sensor
+
+    return associated_sensors
+
+def get_sensor_mapping(site):
+    """
+    Returns the sensor mappings for a 
+    given site from the seshdash_sensor_mapping table
+    """
+    table_sensor_mapping = get_table('seshdash_sensor_mapping')
```
Comment (jlark) [inline api.py]: put this table in the config

File: api.py [hunk 3]
```diff
@@ -27,19 +27,18 @@
     ROLLBAR_TOKEN=None,
     LOG_LEVEL='DEBUG',
     ENVIRONMENT='development',
-    TABLE_NAME='seshdash_bom_data_point',
     STATUS_TABLE_NAME='seshdash_rmc_status',
```
Comment (jlark) [inline api.py]: can yo remove status table as well

**Comment by jlark** [comment]
LGTM

**Comment by jlark** [comment]
#37

**Comment by bumi** [comment]
\o/ awesome. thanks for the PR. looks good.

**Pull request merged**